### DISCUSSION TITLE
Switch to a different language with keeping progress.

### DISCUSSION THREAD

**Discussion by szalejot** [axelboc/anki-ultimate-geography#629] [Q&A] [answered]
I've used English version for a while and I have a review history on the most of the cards.
I wanted to switch to Polish version, but it doesn't update notes. It creates new ones. In the CrowdAnki JSON the same note has different guids for different languages.
Is there a way to switch to another language easily with keeping the current cards review history?

**Comment by aplaice** [comment, marked as answer]
That behaviour is to allow people to have multiple language versions in parallel.

<hr/>

However, if you want to be able to switch, then you can import this deck:
[Ultimate_Geography_v5.2_PL_EXTENDED_guids.zip](https://github.com/anki-geo/ultimate-geography/files/14368931/Ultimate_Geography_v5.2_PL_EXTENDED_guids.zip)

(It has the same set of GUIDs as the English version.)

(First make a backup, then delete the newly imported Polish deck (the one with different GUIDs), and then import this.)

I hope this works without issues! (In testing it worked.)

**Reply by aplaice** [reply]
In case people wanting to switch to other languages are interested, here's a zip of all the decks (i.e. not just Polish Extended), with "English" GUIDs (i.e. these will all replace each other if imported one over the other):

[ug_with_en_guids.zip](https://github.com/anki-geo/ultimate-geography/files/14369072/ug_with_en_guids.zip)

The media files are all the same and are in the top-level `media/` directory (so before importing, move `media/` into the desired `Ultimate Geography...` directory).

(It's a matter of: `sed '20,$ s/guid:[a-z][a-z]/guid/'   -i recipes/source_to_anki.yaml` and `pipenv run brain_brew run recipes/source_to_anki.yaml`.)

(Note that I've only tested the Polish extended version.)

**Reply by szalejot** [reply]
Thank you for help 😌
